### ISSUE TITLE
[FEATURE REQUEST]

### ISSUE THREAD

**Issue by alexdolli** [PTDefender/Welcome#64]
Could you please change the minimum percentage for Max-Rebuy Percentage? 

For my use case, I need to be able to go lower than the minimum of 10 % PTDF offers now. I assume, it would be a relatively easy change. 

And yes, I understand that it takes forever and a million rounds to get out of a bag. But if the option is there, it won't bother anyone or force them to use it. But it would help me tremendously in my use case scenario.  

Pretty please, and thank you in advance!

**Comment by F1power** [comment]
I fully agree. Defender eats up way to much balance. Rebuy should have a zero setting. Defender should put just above Dust in DCD and put the rest in pending and let PT sell it then repeat
